chore(js-react-native): add packages should be removed in migration guide (#6685)
aws-amplify/docs: src/fragments/lib/troubleshooting/common/upgrading.mdx

- DataStore
- In-App Messaging

If you previously listed `@aws-amplify` namespaced packages for the above categories as dependencies in your `package.json`, you will need to remove them to avoid dependency duplication.

If you would like to use Geo, Predictions, PubSub, or Interactions you will need to install those packages separately. (see [category-specific migration instructions](/[platform]/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#4-upgrade-category-usage))

</Block>

<Block name="React Native">

> In v6, the following packages have been replaced with `@aws-amplify/react-native` and should be removed from your `package.json`.

Reviewer: ```suggestion
In v6, the following packages have been replaced with `@aws-amplify/react-native`, so they should be removed from your `package.json`.
```

> In v6, the following packages have been replaced with `@aws-amplify/react-native`, so they should be removed from your `package.json`.

Reviewer 2: ```suggestion
In v6, the following packages have been replaced with `@aws-amplify/react-native` and should be removed from your `package.json`.
```

Reviewer: That is why we need grammar checks instead of people in the team doing the parts of speech check 🙂

Reviewer: TBH Grammar check is not capable of detecting this nit. Super Glad to have on out team @jimblanc ❤️

Reviewer: But I like the original sentence it made some reasoning for doing why. I would opt for the first one.
